pkg/azurelustreplugin/entrypoint.sh

```diff
@@ -56,27 +62,27 @@ echo "$(date -u) Command line arguments: $@"
 
 if [[ "${installClientPackages}" == "yes" ]]; then
   kernelVersion=$(uname -r)
-  
+
   echo "$(date -u) Installing Lustre client packages for OS=${osReleaseCodeName}, kernel=${kernelVersion} "
 
   curl -sL https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor | tee /etc/apt/trusted.gpg.d/microsoft.gpg > /dev/null
-  echo "deb [arch=amd64] https://packages.microsoft.com/repos/amlfs/ ${osReleaseCodeName} main" | tee /etc/apt/sources.list.d/amlfs.list
+  echo "deb [arch=amd64] https://packages.microsoft.com/repos/amlfs-${osReleaseCodeName}/ ${osReleaseCodeName} main" | tee /etc/apt/sources.list.d/amlfs.list
   apt-get update
 
   # Install Lustre client module
-  lustreClientModulePackageVersion=$(apt list -a lustre-client-modules-${kernelVersion} | awk '{print $2}' | grep ^${requiredLustreVersion} | sort -u -V | tail -n 1  || true)
+  lustreClientModulePackageVersion=$(apt list -a ${pkgName} | awk '{print $2}' | grep ^${kernelVersion} | sort -u -V | tail -n 1  || true)
```

Comment:
Nit: This is used in lustre-client-modules-${kernelVersion}=**$lustreClientModulePackageVersion** before, but it's useless now as meta-package version is kernel-version now.

Reply:
Check removed.  PR updated with change squashed.